minecraft_servers/eindino/manifest.json

```diff
@@ -6,6 +6,12 @@
     "de",
     "en"
   ],
+  "server_wildcards": [
+    "eindino.de",
+    "eindino.eu",
+    "magicalsmp.de",
+    "gamingcode.dev"
+  ],
```

Comment:
These are not valid wildcards. See the documentation for more information https://github.com/LabyMod/server-media/blob/master/docs/Manifest.md#available-keys

Reply:
removed it